README.md

```diff
@@ -1,68 +1,72 @@
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## How to Use
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## Known Issues
 
-### Code Splitting
+## Running Tests
+`npm test` - Launches the test runner in the interactive watch mode.
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+## Screenshots
+<!-- ![title](img_url) -->
+
+## How to Contribute
+To contribute, see the setup instructions.
+* [Open Issues](https://github.com/JoelSmith123/git_wired_fe/projects/1)
+* Create a new branch describing the feature. If you close an issue, include it's number in the branch name.
+* Please describe all changes in the Pull Request (to `Master`), and all relative issue cards/actions.
+* Please use the Pull Request Template as the baseline for communication - feel free to add more!
+* Please update the `README` if anything is affected.
+
+
+#### Core Unfinished Features
+[FE Project Board](https://github.com/JoelSmith123/git_wired_fe/projects/1)
+[BE Project Board](https://github.com/hbellows/git_wired_be/projects/1)
+FE Core Features:
```

Comment:
TO DO - 49-51 appear on the same line - add spacing or something to improve format.